### ISSUE TITLE
parameter dictonaries

### ISSUE THREAD

**Issue by aradermacher** [BAMresearch/amworkflow#26]
should we use always pint parameters?

**Comment by YuXHe15** [comment]
> should we use always pint parameters?

I think it would be a good idea to adopt pint to all of our parameters. The only flaw I met till now is that, although there is a package called "pint-pandas",  it does not work well when you want use pint with pandas.

**Comment by aradermacher** [comment]
That's right, I had some problems there. But do we use pandas that much in this framework?
I will look into that as soon as I have some free time
